### ISSUE TITLE
[A11Y] Consider using a GHA to provide some automated checks for accessibility (axe/Pa11y etc.)

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#6411]
The AXE project has been mentioned at work: https://github.com/dequelabs/axe-core ...and they seem to have a GHA available: https://github.com/dequelabs/axe-linter-action

I had concerns about cost and the fact that our project/community has small budget etc., and the following was mentioned in one of the accessibility-focused Slack channels (I haven't fact-checked any of that - just mentioning it since I found it interesting/useful):
>You can use the open source version, specially in the UI interface of you CMS. The also have a really cool extension for Chrome with the same offering.
>
> It is the same open source project google build into the lighthouse. Google actually started with building something for them self, but then the decided to stop when Axe made their tool open source, and just include that instead

In general though, I would like us to explore using a GHA to automate some a11y checks. It won't be everything and we'll still need to do manual checks for various things, but it will be something.

These resources/projects might also be worth having a look at:
https://github.com/canaxess/a11y-github-actions
https://pa11y.org

This was also mentioned: https://accessibilityinsights.io/docs/web/overview
> If you want a bit more than automated checks only I will recommend Accessibility Insights for Web. It also builds on the Axe Core checks for automated checks but will also guide you step by step through all the manual checks which you may not get from the other tools. It is a free Chrome Extension and also includes a lot of good documentation.
